.storybook/preview.tsx

```diff
@@ -14,7 +14,9 @@ import tailwindConfig from '../tailwind.config'
 
 import '../src/styles/index.css'
 
-ReactGA.initialize('G-65N1S3NF5R')
+if (process.env.STORYBOOK_NODE_ENV === 'production') {
```

Comment:
IMO: 細かいんですが、変数に出してしまいたい。

```suggestion
const isProduction = process.env.STORYBOOK_NODE_ENV === 'production'

if (isProduction) {
```